pep-0590.rst

```diff
@@ -121,13 +133,29 @@ For getting the actual number of arguments from the parameter ``n``,
 the macro ``PyVectorcall_NARGS(n)`` must be used.
 This allows for future changes or extensions.
 
+Finalizing the API
+------------------
+
+The underscore in the names ``_PyObject_Vectorcall`` and
+``_Py_TPFLAGS_HAVE_VECTORCALL`` indicates that this API may change in minor
+Python versions.
+When finalized (which is planned for Python 3.9), they will be renamed to
+``PyObject_Vectorcall`` and ``Py_TPFLAGS_HAVE_VECTORCALL``, with the former
+names provided using macros.
```

Comment:
I would make `with the former names provided using macros` more clear and write something like `the old underscore-prefixed names will remain available as aliases` or something.